services/src/performance-tests/README.md

```diff
@@ -0,0 +1,8 @@
+**Monitoring**
+
+[Sampler](https://github.com/sqshq/sampler) is a tool for shell commands execution, visualization and alerting.
+Configured with a simple YAML file.
+
+If there is a way to get a metric using shell command - then it can be visualized with Sampler momentarily. 
+
```

Comment:
Any quick commands to make this running?

Reply:
`sampler -c config.yml`